src/pixel/state.js

```diff
@@ -185,9 +206,14 @@ export function StateWrapper (state) {
   /**
    * @type {State}
    */
+  let stateWithAdditionalFields = {}
   let _state = {}
   if (state) {
-    _state = _safeFiddle(state)
+    if ('gdprApplies' in state) {
+      const gdprApplies = state.gdprApplies
+      stateWithAdditionalFields = { ...state, n3pc: gdprApplies, n3pc_ttl: gdprApplies, nbakers: gdprApplies }
```

Comment:
gdprApplies evaluates to 1 even if the value if not a boolean - e.g. s ? 1 : 0 with s="a" would evaluate to 1. We should compute the values of n3pc, n3pc_ttl and nbakers so that it always matches the value of gdprApplies: gdprApplies == 0 => all three are false, gdprApplies == 1 => all three are true.

Reply:
Initially gdprApplies is a boolean and during initializing the state will be converted to 1 or 0 in "gdpr" parameter